### PR TITLE
Removed workload.sas.com/node from code/doc.

### PR DESCRIPTION
**Pull request by thpang**
Also added in required system nodepool default label: `"kubernetes.azure.com/mode" = "system"`

### CODE REVIEW

**Comment by thpang** [comment]
CONFIG-VARS.md was modified incorrectly by VSCode plugin. Recalling PR.

**Comment by thpang** [comment]
Ok, seems the index update was needed. Re-opening.

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Supported configuration variables are listed in the table below.  All variables
     - [Default Nodepool](#default-nodepool)
     - [Additional Nodepools](#additional-nodepools)
   - [Storage](#storage)
-    - [storage_type=standard - nfs server VM](#storage_typestandard---nfs-server-vm)
-    - [storage_type=ha - Azure NetApp](#storage_typeha---azure-netapp)
+    - [NFS Server VM (only when `storage_type=standard`)](#nfs-server-vm-only-when-storage_typestandard)
```
Comment (enderm) [inline docs/CONFIG-VARS.md:17]: Maybe leave the stuff in the brackets out of the TOC, so the TOC is cleaner i. e.

 - [NFS Server VM](#nfs-server-vm-only-when-storage_typestandard)
 - [Azure NetApp](#azure-netapp-files-only-when-storage_typeha)

**Review by enderm — APPROVED** [review]
lgtm - one optional nitpick in the POC

**Pull request merged**